### DISCUSSION TITLE
About the sitelog sync for GNSS nodes

### DISCUSSION THREAD

**Discussion by PierreS-alpha** [IPGP/webobs#122] [Ideas]
Hi,

I have 3 very small questions/suggestions for the implementation of the sitelog sync:

1. I do not remember why we decided to implement the new `IGS_9CHAR` variable field in the _Geographic location_ section. It looks more relevant to me to  set in in the _Names and Description_, no?
4. Moreover, maybe `GNSS_9CHAR` might be more explicit for non-geodetist  programmers,  rather than `IGS_9CHAR`, no?  
6. And last point, regarding  the WebObs internal naming conventions which variable name is the best:  `GNSS_9CHAR` or `GNSS9CHAR`? Maybe the underscore is not really necessary.

Thanks,
Pierre

**Comment by beaudu** [comment]
Hi Pierre,
Yes a name like `GNSS_9CHAR` is probably more appropriate. I suggest to keep the underscore to keep consistency with other node's variables (.cnf).
For your first comment, the *Names and Description* section is not functional (field content has no effect than for display purpose). That's why I suggest to put it in the *Geographic Location* section (or at least in the right part of the form).
François.

**Reply by PierreS-alpha** [reply]
fine, thanks for your prompt answer!

**Reply by beaudu** [reply]
we might also rename the section *Geographical Location* with something more generic, like *Site Metadata* or something evoking the site log (that includes the geographic location)...

**Comment by PierreS-alpha** [comment]
Another small question: do we have already some procedures (functions, piece of scripts...) to automatically modify/update the feature text files?

**Reply by beaudu** [reply]
Edition tools related to node's features are:

- **nedit.pl** : manual edition of each feature file;
- **formNODE.pl** and **postNODE.pl** : list of features in the variable `FILES_FEATURES`,
- **postNODE.pl** : list of cross links between feature/nodes (file pointed by `FILE_NODES2NODES` in **NODES.rc**).
